source/includes/_events.md

```diff
@@ -1,5 +1,22 @@
 # Events
 
+Every call to action is represented in the API as an event.  Events have related resources such as a contact or a location.
```

Comment:
We can still have that discussion. 

The reason I used the term "event" to encapsulate onsite and phone CTAs was that term "call to action" has unusual pluralization rules.  With most words in English, you add an "s" and you're done, but when you have more than one call to action, you have "calls to action."

Inside the app, we can configure rails to respect this way of pluralizing the term, but then we also need to have this unusual pluralization present in our endpoint names.

I figured it'd be a PITA to do stuff like troubleshoot why the calls to the API wasn't working in a situation where it turned out that there should (or shouldn't) be an `s` in the middle of the url.  We're mostly accustomed to checking at the end of an URL to verify that there is (or isn't) an `s`.  Long story short, I wanted to make it as easy as possible for API consumers to deal with this API.

One alternative would be to just call "call to actions" "actions", but I choose not to since the term "action" is already pretty loaded in the web dev world.  I didn't want to cause confusion between the concept of HTTP actions and calls to action.

I agree that "events" doesn't really sound right when we're talking about phone-based calls to action.  If you have another suggestion let me know.

What do you think about just calling them "CTAs"?  API Consumers would head to `/ctas` for a list or pass a uuid along to `/cta/` to get a specific CTA.  Thoughts?

This wouldn't be an especially difficult thing to change at this stage in the game.